### ISSUE TITLE
Update workshop materials based on feedback from April

### ISSUE THREAD

**Issue by kmcdono2** [Living-with-machines/MapReader#417]
Using feedback from workshop (Data/Culture Sharepoint Documens --> LwM workshop reporting), these are things we will adapt for the June 2024 workshop.

- [ ] #423
- [ ] #439
  - [x] split people with PCs/Macs into different tables
  - [ ] organise pre-workshop zoom to trouble shoot install, depending on audience?
  - [ ] #421
  - [x] clarify what prep to do in advance: read MapReader paper,  programming historian lessons about CV/ML, python cheat sheet, juptyter notebooks, etc.
- [ ] #422
  - [ ] how to stay in touch
  - [ ] how to find new models and share yours
  - [ ] how to share datasets
- [ ] #440
- [x] #420
- [x] early on in workshop, have people look at felt map of railspace data to talk about nature of outputs, error, success, etc. so they understand the potential
- [x] hyperlinks to relevant docs within notebooks
- [x] explain training, validation sets
- [x] explain epochs, loss and how this allows you to fine-tune your model
- [ ] talk about building footprint task
- [x] encourage teamwork/propose shared annotation task
- [ ] comment each command in version of notebook that is for python beginners
- [ ] have different versions of notebooks: completely commented and filled in vs. bare bones
- [x] more guidance about annotating, e.g. explain typical devisions around guideline development, importance of "none" label
- [ ] reduce CV/ML intro & model/train slides to what you need to know specifically for MapReader 
- [ ] lunch 45 min, not 60?
- [ ] #441
- [x] notebook for working with text spotting task output
- [ ] for free time: guidance for helping people do something in QGIS with their output file
- [ ] consider working on IIIF as input to make it easy for people to grab those (non-georef) maps

**Comment by rwood-97** [comment]
Also need to address:
- [ ] #411 
- [ ] #406 
- [ ] #408 
- [ ] #409 
- [ ] #410

**Comment by rwood-97** [comment]
Before starting notebooks:
- [ ] explain the comment and how to fill in commented sections
- [ ] explain passing variables to functions

Suggestion to perhaps include in part2 notebook:
After the inference on the first GLASGOW sheet, to run inference on another map sheet (or include this as an exercise - it just involves changing the sheet ID in a couple of places). One participant was really pleased to see how good the results were on other sheets.

**Comment by rwood-97** [comment]
@katie can I close this ticket since all the other main unfinished points have their own issues now
